src/python/grpcio/grpc/aio/_channel.py

```diff
@@ -207,7 +207,7 @@ def __call__(
             call = InterceptedUnaryStreamCall(
                 self._interceptors,
                 request,
-                deadline,
+                timeout,
```

Comment:
Can we also delete the calculation of deadline above this, since it is now unused? Here and in the other two arities?